### PR TITLE
Attendee Backend

### PR DESCRIPTION
**Pull request by alicesf2**
Refs #277

### CODE REVIEW

File: backend/src/api/attendees.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    const attendee = await Attendee.findOneAndUpdate(
+      { netId },
+      { $set: fieldsToUpdate },
+      { new: true }
+    )
+    const ret = attendee
+      ? {
+          code: 200,
+          message: 'Attendee Updated Successfully',
+          success: true
```
Comment (alan-fang) [inline backend/src/api/attendees.js]: A random comment that is not necessarily related to your PR.. is there a reason why we have a `code` and a `success` field? Wouldn't a `code` field of 200 suggest that the response is a success? Or am I misinterpreting the meaning of `success` here?

File: backend/src/models/event.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ const Event = new mongoose.Schema({
   endTime: { type: String, required: true },
   location: { type: String, required: true },
   description: { type: String, required: true },
-  attendees: { type: [AttendeeSchema], required: true }, // subdocument
+  attendees: { type: [mongoose.Schema.Types.ObjectId], required: true },
+  lateAttendees: { type: [mongoose.Schema.Types.ObjectId], required: true },
```
Comment (alan-fang) [inline backend/src/models/event.js]: omg lol

File: backend/src/api/attendees.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+// create a new attendee
+router.post(
+  '/',
+  errorWrap(async (req, res) => {
+    const data = req.body
+    const newAttendee = new Attendee({
+      name: data.name,
+      netId: data.netId,
+      email: data.netId + '@illinois.edu',
```
Comment (alan-fang) [inline backend/src/api/attendees.js]: hmmmmm the functionality I'm working on (Workspaces) is supposed to allow us to give the tool to other student groups and schools for them to use. So it's not necessarily true that the email should be suffixed with '@illinois.edu'. Similarly, the field 'netId' probably is not a great descriptor either (although this is a non-issue and more of a naming nit). I recommend dropping the email column entirely or having emails be an explicit entry.

Comment (alicesf2) [reply backend/src/api/attendees.js]: ok

File: backend/src/models/event.js (hunk 2 above)
Comment (alicesf2) [reply backend/src/models/event.js]: ok

**Comment by tko22** [comment]
📸🐣

**Pull request merged**